template/yarn.lock

```diff
@@ -5579,11 +5579,16 @@ source-map@^0.5.0, source-map@^0.5.3, source-map@^0.5.6:
   resolved "https://registry.yarnpkg.com/source-map/-/source-map-0.5.7.tgz#8a039d2d1021d22d1ea14c80d8ea468ba2ef3fcc"
   integrity sha1-igOdLRAh0i0eoUyA2OpGi6LvP8w=
 
-sourcegraph@23.0.1, sourcegraph@^23.0.1:
+sourcegraph@23.0.1:
   version "23.0.1"
   resolved "https://registry.yarnpkg.com/sourcegraph/-/sourcegraph-23.0.1.tgz#715fcf4129a6d94bc3bfd2740d9c706ae6357ffe"
   integrity sha512-4We7zqhOagOVxNFdS6/xT/Crhb0Arw/9ytGBu8JuHfjo5yjMtcUYt22kZyu2TaPHXwyPW9muUi1eKSFA6Qg4lw==
 
+sourcegraph@^23.1.0:
```

Comment:
The package is duplicated now, which is problematic since conceptually there can only be one instance

Reply:
Missed this. Thanks fro the catch.